src/ocm/components/AddHosts/types.ts

```diff
@@ -37,5 +37,16 @@ export type OcmClusterType = {
     url: string;
   };
 
+  metrics?: {
+    nodes: [
+      {
+        compute?: number;
+        infra?: number;
+        master?: number;
+        total?: number;
+      },
+    ];
```

Comment:
OMG, this is not always an array!!!

![image](https://user-images.githubusercontent.com/77008341/152181243-fb0a4bc8-bcfe-49a0-bd48-874c1ace78d7.png)